src/LercLib/Lerc2.h

```diff
@@ -118,10 +118,11 @@ class Lerc2
   bool Decode(const Byte** ppByte, size_t& nBytesRemaining, T* arr, Byte* pMaskBits = nullptr);    // if mask ptr is not 0, mask bits are returned (even if all valid or same as previous)
 
 private:
-  static const int kCurrVersion = 4;    // 2: added Huffman coding to 8 bit types DT_Char, DT_Byte;
-                                        // 3: changed the bit stuffing to using a uint aligned buffer,
-                                        //    added Fletcher32 checksum
-                                        // 4: allow nDim values per pixel
+  // 2: added Huffman coding to 8 bit types DT_Char, DT_Byte;
+  // 3: changed the bit stuffing to using a uint aligned buffer,
+  //    added Fletcher32 checksum
+  // 4: allow nDim values per pixel
+  static int CurrentVersion() { return 4; }

```

Comment:
since this is private and static, would it make more sense to put this just in the lerc2.cpp in an anonymous namespace?

Reply:
No, it should be in the .h file to be visible. Also it is easy to make it public if needed by just moving it up into the public section. And it is inline, so there is no function call.